### ISSUE TITLE
Bug with EXP tracker (Defence)

### ISSUE THREAD

**Issue by erikmukk** [runelite/runelite#6396]
Please check if your issue is not a duplicate by [searching existing issues](https://github.com/runelite/runelite/search?type=Issues)

**Describe the bug**
So the EXP tracker is showing like I have negative EXP and 0 LVLs in Defence. And I need to do 2.1B actions for level

**To Reproduce**
No idea, it just happened to me. 
See screenshot down below for explanation.

**Expected behavior**
EXP tracker should work properly like it does for other skills.

**Screenshots**
https://gyazo.com/dd2de963d5897a76f918f3b1447f6adc

**Environment (please complete the following information):**
 - OS: Windows
 - RuneLite version: 1.4.25
 - Launcher version: 1.6.0

**Additional context**
Add any other context about the problem here, e.g. logs. Your client logs can usually be found in
[application.log](https://github.com/runelite/runelite/files/2563116/application.log)

**Comment by Karn** [comment]
A similar bug is seen when the XP globes are still up and the trackers are reset. The XP globes then show -MAX_INT as the XP remaining. I think it might be related but I'm not entirely positive.

**Comment by critzd** [comment]
Can you elaborate as to what you were doing at the time? Did this only occur for a brief moment? Were you training defence at all? We need more information to properly investigate it.

**Comment by raiyni** [comment]
It happens when you have boosted skills and killing things.

**Comment by erikmukk** [comment]
I was killing Wall beasts in Lumbridge swamp dungeon. I realised that defence exp tracker was showing these things, when I reached 60 attack (I was wielding Brine sabre). Then I switched my combat style style to train strength (earlier I was training attack).

I had no boosted stats what so ever.

Now I have closed the client so I can not check if it is still showing it incorrectly.

Tomorrow when I log in, I will see if it is still messed up.
